source/uwp/SharedRenderer/lib/Util.h

```diff
@@ -225,3 +225,12 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering
 }
 
 std::string ExtractSvgDataFromUri(winrt::Windows::Foundation::Uri const& imageUrl);
+
```

Comment:
make sure the source this is copied from is removed in AdaptiveImageRenderer.cpp we don't want to duplicate.